docs/hardware/devices/canary/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+id: canaryone
+title: CanaryOne
+sidebar_label: Canary
+sidebar_position: 1
+---
+
+
+The CanaryOne was built by Canary Radio Company from the ground up to run Meshtastic in austere environments as an affordable tactical networking solution.
+
+CanaryOne custom PCB and case were designed and are assembled and manufactured in the United States.
+
+The custom PCB board supports jlink debugging, with intelligent electrical components sourced from the United States and Western Europe. 
+
+![CanaryOneFront](/img/hardware/canary-one/front.webp)
+
+## Specifications
+
+- **MCU**
+  - NRF52840 (Bluetooth)
+    - Bluetooth 5.0
+- **LoRa Transceiver**
+  - Semtech SX1262
+- **Frequency options**
+  - All Mesthastic options.
```

Comment:
Typo "Mesthastic"

Reply:
In my find all, found another spot where mesthastic was mispelled, so fixed that too.  We could consider implementing cspell.  I might add that in another PR.

Reply:
it has proven to be a very difficult word to spell properly.